packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -294,6 +294,12 @@ $form-input-focus--boxShadow              : 0 0 0 2px $form-input-focus;
     & + & {
         margin-top: spacing(x2);
     }
+
+    &:hover {
+        .c-formField-field {
```

Comment:
Looking at the video, this means the hover applies when hovering over any part of the outer wrapper?

This looks a bit strange to me – what's the reason for changing this? (would be the sort of change I'd user test to see if users think it's a bit confusing)

Reply:
Notice you've normalised it actually, so it's not a new addition. May have already checked, but do we know if it should be that the background always appears (when hovering labels) or should we have it so that it never changes the bg colour (so make it consistent the opposite way, if that makes sense).